plugins/aja/aja-source.cpp

```diff
@@ -184,6 +184,63 @@ static inline uint64_t get_sample_time(size_t frames, uint_fast32_t rate)
 	return os_gettime_ns() - samples_to_ns(frames, rate);
 }
 
+static bool aja_source_check_transfer_audio_dma(
```

Comment:
I'm using the `aja_source_` function prefix for functions which are callbacks to the OBS plugin API. See here:
https://github.com/obsproject/obs-studio/blob/0ed34427e111e9d899b3a6ad708bc98f3139b0f3/plugins/aja/aja-source.cpp#L1164
Could you just rename the new functions without that prefix?